src/main/kotlin/us/ihmc/build/IHMCBuildPlugin.kt

```diff
@@ -12,11 +12,18 @@ import org.gradle.api.tasks.Delete
 import org.gradle.kotlin.dsl.create
 import org.gradle.plugins.ide.eclipse.EclipsePlugin
 import org.gradle.plugins.ide.idea.IdeaPlugin
+import us.ihmc.cd.AppExtension
+import us.ihmc.cd.RemoteExtension
 
 class IHMCBuildPlugin : Plugin<Project>
 {
    override fun apply(project: Project)
    {
+      // add deploy task
```

Comment:
Perosnally I would just remove both these comments, but this one is wrong. The deploy tasks are added by the user, which utilize these two extensions. The app extension just allows a simpler way to have multiple mains coorrespond to CreateStartScripts

Reply:
(Both of these can be useful for things other than deploying as well)

Reply:
Yeah, I just missed these. Removed.